docs(config): add use cache (#1571)
nestjs/docs.nestjs.com: content/techniques/configuration.md

> info **Hint** The `ConfigType` is exported from the `@nestjs/config` package.

#### Use cache

> Accessing `process.env` is slow. To increase the performance of the `ConfigService.get` method for variables stored in the `proccess.env` set the `cache` property of the options object we pass to the `ConfigModule.forRoot()`.

Reviewer: ```suggestion
As accessing `process.env` can be slow, you can set the `cache` property of the options object passed to `ConfigModule.forRoot()` to increase the performance of `ConfifService.get` when it comes to variables stored in `process.env`
```
